app/views/diary_entry/rss.rss.builder

```diff
@@ -21,7 +21,7 @@ xml.rss("version" => "2.0",
         xml.link url_for(:action => "view", :id => entry.id, :display_name => entry.user.display_name, :host => SERVER_URL)
         xml.guid url_for(:action => "view", :id => entry.id, :display_name => entry.user.display_name, :host => SERVER_URL)
         xml.description entry.body.to_html
-        xml.author entry.user.display_name
+        xml.dc :creator, entry.user.display_name
```

Comment:
As the commit comment says, the author tag is reserved for email addresses.

http://www.rssboard.org/rss-profile#namespace-elements-dublin-creator

"The value of the dc:creator element is less restrictive than the author element, which must contain an e-mail address. Publishers often rely on dc:creator to credit authorship without revealing e-mail addresses in a form that can be exploited by spammers."